core/src/wallet/ripple/explorers/api/RippleLikeTransactionParser.cpp

```diff
@@ -131,10 +131,11 @@ namespace ledger {
                     _transaction->confirmations = value.toUint64();
                 } else if (_lastKey == "ledger_index") {
                     RippleLikeBlockchainExplorer::Block block;
-                    BigInt valueBigInt = BigInt::fromString(number);
-                    block.height = valueBigInt.toUint64();
+                    block.height = value.toUint64();
```

Comment:
That change is not linked to `DestinationTag`. Why is it needed?

Reply:
Just a small cleanup. We already had `BigInt value = BigInt::fromString(number);` declared outside the `if` block.